packages/utils/src/global.ts

```diff
@@ -72,15 +72,6 @@ export const GLOBAL_OBJ: InternalGlobal =
   })() ||
   {};
 
-/**
- * Safely get global scope object
- *
- * @returns Global scope object
- */
-export function getGlobalObject<T>(): T & InternalGlobal {
```

Comment:
For now let's deprecate this just in case, and we can remove in v8 - I know we don't have to follow semvar in utils, but `getGlobalObject` might be more commonly used than other functions, and doesn't cost us much to just wait.

Reply:
Yep, better idea!